app/src/main/java/ai/elimu/vitabu/util/ReadingLevelExtension.kt

```diff
@@ -4,10 +4,10 @@ import ai.elimu.model.v2.enums.ReadingLevel
 
 fun ReadingLevel?.toSpeechRate(): Float {
     return when (this) {
-        ReadingLevel.LEVEL1 -> 0.4f
-        ReadingLevel.LEVEL2 -> 0.6f
-        ReadingLevel.LEVEL3 -> 0.8f
-        ReadingLevel.LEVEL4 -> 1.0f
-        null -> 1.0f
+        ReadingLevel.LEVEL1 -> 0.2f
+        ReadingLevel.LEVEL2 -> 0.3f
+        ReadingLevel.LEVEL3 -> 0.4f
+        ReadingLevel.LEVEL4 -> 0.5f
+        null -> 0.5f
```

Comment:
@tuancoltech I did a manual test to see how many Words Per Minute (wpm) this setting would be, and I got results looking something like this:
```
LEVEL1: 120 words per minute
LEVEL4: 175 words per minute
```

This speed is much higher than expected for the 4 levels:

![](https://github.com/user-attachments/assets/556cb879-187c-4087-ae08-8fe2206bc558)

After 6 months of using our software, we expect a student to have become a fluent reader, which means a reading speed of at least 50 words per minute.

So I suggest changing the values to these values:

```suggestion
        ReadingLevel.LEVEL1 -> 0.04f
        ReadingLevel.LEVEL2 -> 0.08f
        ReadingLevel.LEVEL3 -> 0.16f
        ReadingLevel.LEVEL4 -> 0.40f
        null -> 0.40f
```

Reply:
@jo-elimu Agree.
Could you test again now?